webbeans-junit5/src/main/java/org/apache/openwebbeans/junit5/Cdi.java

```diff
@@ -80,6 +80,11 @@
      */
     boolean disableDiscovery() default false;
 
+    /**
+     * @return {@code true} to enable JUnit parameter resolution with CDI beans or {@code false} otherwise.
+     */
+    boolean injectParameters() default true;
```

Comment:
as suggested by @rmannibucau

Reply:
Hi @philippkunz , I was more thinking about another annotation cause it can't be global.
Here is an example:

    @Test void test( MyCdiBeanInjectedByJUnitExtension1 b2, MyCdibeanInjectedByCdiExtension b2 ) {}

If cdi extension runs first it does not work so we must have:

    @Test void test( @Cdi.SkipInject MyCdiBeanInjectedByJUnitExtension1 b2, MyCdibeanInjectedByCdiExtension b2 ) {}

(or equivalent API). Note that making SkipInject a qualifier (added from an implicit extension in the extension) makes it trivial to impl ;).

Hope it makes sense

Reply:
Although I can see the case in a theoretical sense, I have some difficulty to imagine a real situation where another approach would not sufficiently help such as setting `Cdi(disableDiscovery = true)` or listing fewer classes in `@Cdi(classes = ...)` or first of all working with CDI stuff (types, qualifiers, scopes, or `bean-discovery-mode`) or moving the parameter to an `@Inject`-annotated class variable to prevent Cdi to inject such a parameter that should be resolved by another extension.
On the other hand such an annotation to tell Cdi not to resolve a parameter would not do any harm either. In fact, that way the discussion about `getParent` can be worked around.

Reply:
I have several extensions doing a cdi lookup themselves with custom rules. This impl breaks it until I disable the feature. Same happens with framework having an IoC abstraction (often cdi/spring/guice) where the lookup is done internally in the extension - going through the IoC abstraction.
I wouldnt say it is 50% of tests but likely 10% where it can hit and as mentionned supporting it is almost just one qualifier for us so it is worth it. That said I'm happy to do it after your PR is merged if you don't want to do it in this PR too, just let me know the status ;).

Reply:
Honestly I'm afraid I still haven't understood your concern really. To me this sounds like you would expect two injection mechanisms to play together well, which strikes me as quite odd...

Reply:
Do you have a pointer to the mentioned examples with other extensions?

Reply:
When you write qualifier, do you mean CDI `@Qualifier` specifically? Like this?

```
@Qualifier
@Target({ TYPE, METHOD, PARAMETER, FIELD })
@Retention(RUNTIME)
@Documented
public @interface NotResolvedAsCdiBeanIntoJunitParameter {
}
```
and then
```
@Test void someTest(@NotResolvedAsCdiBeanIntoJunitParameter MyService service) {
}
```
and not having any or no otherwise eligible bean class annotated `@NotResolvedAsCdiBeanIntoJunitParameter`

That can be done without having such a qualifier defined in the extension here. It can be defined and used in any project using it. I have the feeling it's according to my current understanding not a general enough concern to have it in the PR here. Does that match your point?